example/export/bundle.js

```diff
@@ -0,0 +1,84 @@
+/*****************************************************************************
+ * Open MCT Web, Copyright (c) 2014-2015, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT Web is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT Web includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+/*global define*/
+
+define(['legacyRegistry'], function (legacyRegistry) {
+    "use strict";
+
+    function ExportTelemetryAsCSVAction(exportService, context) {
```

Comment:
I think it's really nice to see this in one file.  It greatly simplifies understanding.  Yes, you should break functionality into multiple files, but different projects can define their own module structure.  A single file is an incredibly easy to understand "getting started" guide and is very valuable.  

We're not trying to demonstrate how to properly package and separate code-- we're trying to demonstrate how to use a feature.